data/mods/ssb/moves.ts

```diff
@@ -127,6 +127,7 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		},
 		onHit(pokemon) {
 			pokemon.addVolatile('stall');
+			pokemon.m.swapSets();
```

Comment:
```suggestion
			if (source.species.baseSpecies !== 'Aegislash') return;
			pokemon.m.swapSets();
```

Reply:
blame grimAuxiliatrix and/or boat for this

Reply:
oh right i forgot transform exists too lol